translations/harbour-seaprint-de.ts

```diff
@@ -554,33 +554,33 @@
     </message>
     <message>
         <source>Ignore SSL errors</source>
-        <translation type="unfinished"></translation>
+        <translation>SSL Fehler ignorieren</translation>
     </message>
     <message>
         <source>In order to work with self-signed certificates of printers and CUPS instances, SSL errors needs to be ignored.</source>
-        <translation type="unfinished"></translation>
+        <translation>Um mit selbstsignierten Zertifikaten von Druckern und CUBS-Instanzen arbeiten zu können, müssen SSL-Fehler ignoriert werden.</translation>
     </message>
 </context>
 <context>
     <name>SupplyItem</name>
     <message>
         <source>Unknown type</source>
-        <translation type="unfinished"></translation>
+        <translation>unbekannter typ</translation>
     </message>
     <message>
         <source>Unnamed supply</source>
-        <translation type="unfinished"></translation>
+        <translation>unbenannter Toner</translation>
     </message>
 </context>
 <context>
     <name>harbour-seaprint</name>
     <message>
         <source>images</source>
-        <translation type="unfinished"></translation>
+        <translation>Bilder</translation>
```

Comment:
This will appear in the middle of a sentence, but that is ok witth uppercase, right?